src/sql/upgrade-62.sql

```diff
@@ -0,0 +1,7 @@
+-- Make the Builds.jobset_id column NOT NULL. If this upgrade fails,
+-- either the admin didn't run the backfiller or there is a bug. If
+-- the admin ran the backfiller and there are null columns, it is
+-- very important to figure out where the nullable columns came from.
+
+ALTER TABLE Builds
+  ALTER COLUMN jobset_id SET NOT NULL;
```

Comment:
Drop the `project` and `jobset` columns?

Reply:
I think we should eventually, but I'm a bit afraid to do that in this PR since it makes it basically impossible to undo if there is a mistake. Also, it means we need to be 100% certain every read path is updated. I think that is a good improvement to make for a follow-up PR, maybe within a few weeks of this being deployed. Sound okay?